### PR TITLE
Add gfsplc

### PR DESCRIPTION
**Pull request by jyin999**
## Motivation and Context

Dan Flath required to add GFS fault setpoint to readable through EPICS. Alex required to automatically turn on GCC and PIP after power outage. The modification allow user to turn on Auto recover function from EPICS GUI. 

## How Has This Been Tested?
Tested following the confluence page. 
image

## Where Has This Been Documented?
Documented following the confluence page. 

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/869-added_GFS_fault_setpoint,_GCC,_PIP_autoOn_and_countdown_timer.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (klauer) [inline docs/source/upcoming_release_notes/869-added_GFS_fault_setpoint,_GCC,_PIP_autoOn_and_countdown_timer.rst:30]:
```suggestion
- jyin999
```

You are the contributor here

**Review by klauer — COMMENTED** [review]
You didn't fill in any of the check boxes in the pull request template - did this work interactively, at least?

File: docs/source/upcoming_release_notes/869-added_GFS_fault_setpoint,_GCC,_PIP_autoOn_and_countdown_timer.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+-GFS, GCC, PIP
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/869-added_GFS_fault_setpoint,_GCC,_PIP_autoOn_and_countdown_timer.rst:14]:
```suggestion
- Added GFS fault setpoint, GCC, PIP auto-on and countdown timer
```

**Comment by ZLLentz** [comment]
PVs look good when I compare with the vacuum library

**Comment by ZLLentz** [comment]
@jyin999, are the new IOC PVs deployed to test this on?

**Comment by jyin999** [comment]
> @jyin999, are the new IOC PVs deployed to test this on?

Yes. 

> You didn't fill in any of the check boxes in the pull request template - did this work interactively, at least?

Oh. I thought the checklist is for reviewer.

**Comment by jyin999** [comment]
> @jyin999, are the new IOC PVs deployed to test this on?

New IOCs have these PVs and they are running.

**Review by klauer — APPROVED** [review]
OK, with your successful local testing and travis passing, I think this is good.

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
Merged + deploying to dev

**Pull request merged**